pow-migration/Cargo.toml

```
@@ -74,6 +74,7 @@ build-data = "0.2"
 serde_json = "1.0"
 
 nimiq-test-log = { workspace = true }
+nimiq-web-client = { workspace = true, default-features = false, features = ["primitives"] }
```

The  `default-features = false` is inherited from the workspace
```suggestion
nimiq-web-client = { workspace = true, features = ["primitives"] }
```